barretenberg/cpp/src/barretenberg/eccvm/eccvm_flavor.hpp

```diff
@@ -293,72 +293,10 @@ class ECCVMFlavor {
 
         auto get_to_be_shifted() { return ECCVMFlavor::get_to_be_shifted<DataType>(*this); }
         auto get_shifted() { return ShiftedEntities<DataType>::get_all(); };
+        auto get_precomputed() { return PrecomputedEntities<DataType>::get_all(); };
     };
 
   public:
-    /**
```

Comment:
large diff here mostly coming from the need to move some class defs around to accommodate ProverPolynomials owned by ProvingKey